cmd/goodman/main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 				}
 			}()
 			for retries := 5; retries > 0; retries-- {
-				runner, err := goodman.NewRunner("Hooks", hookServerInitalPort)
+				runner, err := goodman.NewRunner("HooksRunner", hookServerInitalPort)
```

Comment:
I'm guessing this was the change that fixed the timeouts correct?

I was hoping to introduce a change that threw a more obvious error instead of experiencing timeouts.  Now that I think about it I don't think that would have changed anything.  I'm fine with things either way.